Fatal error: exception Failure("illegal argument found int* expected int in *a")
